.jazzy.yaml

```diff
@@ -0,0 +1,7 @@
+author: Alamofire Software Foundation
+author_url: http://alamofire.org/
+github_url: https://github.com/Alamofire/Alamofire
+module: Alamofire
+output: docs
```

Comment:
We probably want to refactor our Documentation layout with this. Perhaps we can put these docs in `Documentation/jazzy` and move the migration guides `Documentation/Migration Guides`. That will require some Xcode project changes for the guides, but there's no need to add the jazzy docs to Xcode at all.